rtgui/resize.cc

```diff
@@ -81,19 +83,46 @@ Resize::Resize () : FoldableToolPanel(this, "resize", M("TP_RESIZE_LABEL"), fals
     Gtk::HBox* sbox = Gtk::manage (new Gtk::HBox ());
     Gtk::HBox* wbox = Gtk::manage (new Gtk::HBox ());
     Gtk::HBox* hbox = Gtk::manage (new Gtk::HBox ());
```

Comment:
This seems to be based on an older version of `resize.cc`. There have been changes https://github.com/Beep6581/RawTherapee/blob/53bb6e41bb1f4052bb85048fe72499cabcdcb1ec/rtgui/resize.cc#L91

Perhaps you need to rebase?

Reply:
It is in fact based on releases branch, not on dev branch, because the release version is what I use for actual raw processing. I'll try the rebasing as soon as I find the time.

Reply:
@bezierr please base new work on the `dev` branch.